Themes/default/index.template.php

```diff
@@ -325,7 +325,7 @@ function template_body_above()
 	echo '
 	<div id="header">
 		<h1 class="forumtitle">
-			<a id="top" href="', $scripturl, '">', empty($context['header_logo_url_html_safe']) ? $context['forum_name_html_safe'] : '<img src="' . $context['header_logo_url_html_safe'] . '" alt="' . $context['forum_name_html_safe'] . '">', '</a>
+			', !empty($context['current_action']) || !empty($_GET) ? '<a id="top" href="' . $scripturl . '">' : '', empty($context['header_logo_url_html_safe']) ? $context['forum_name_html_safe'] : '<img src="' . $context['header_logo_url_html_safe'] . '" alt="' . $context['forum_name_html_safe'] . '">', !empty($context['current_action']) || !empty($_GET) ? '</a>' : '', '
```

Comment:
I don't believe it matters that much to make the edit here, in fact this change makes the style of the header link inconsistent where in the home page it looks different than in other pages, it should be reversed imho.